commands/codebuild_cli.py

```diff
@@ -5,9 +5,8 @@
 import click
 from boto3.session import Session
 from mypy_boto3_codebuild.client import CodeBuildClient
-
-from utils.aws import check_aws_conn
-from utils.aws import check_response
+from .utils import check_aws_conn
+from .utils import check_response
```

Comment:
I thought we wanted to move everything out to the `utils` folder. 👀

Reply:
i did, but Lyndon created a utils file.  I didnt want utils in two palces

Reply:
okie dokie